application/models/post_test.go

```diff
@@ -416,6 +418,14 @@ func (ms *ModelSuite) TestPost_ValidateUpdate() {
 			},
 			wantErr: false,
 		},
+		{
```

Comment:
If you're so inclined, I think this test (`TestPost_ValidateUpdate`) might be better (e.g. better coverage, smaller fixtures, easier to read) as specific tests on each validator. Sorry I didn't think of it earlier, and no sweat if it's not worth the trouble now.

Reply:
Done